Host check Timeout error handling (#9)
ios-xr/xrd-tools: CHANGELOG.md

It is planned to support 7.8.1 when this version is released.

### v1.0.4 (2022-11-30)

> - `host-check` script now gives a different error when a command times out

Reviewer: ```suggestion
- Indicate when a command times out in the `host-check` script.
```
